### PR TITLE
English lessons

### LINKED ISSUE #100: English lessons policy
This is missing!

### PR DESCRIPTION
**Pull request by aldesantis**
Closes #100.

### CODE REVIEW

**Comment by kennyadsl** [comment]
Still WIP, right?

**Comment by aldesantis** [comment]
@kennyadsl ready now!

File: personal-growth/benefits.md [hunk 1]
```diff
@@ -36,3 +36,6 @@ regarding any of the benefits, feel free to ask!
 - **Internet budget.** If you work remotely, we will pay the costs of your home Internet connection.
   Having a good connection is not only in your interest, but also in the interest of your team
   members, so let us know if you need an upgrade!
+- **English lessons.** If you're not comfortable with your English proficiency, we will happily pay
+  for your English lessons. We recommend [italki](https://www.italki.com/), but you can expense any
+  other platform that helps you learn the language.
```
Comment (kennyadsl) [inline personal-growth/benefits.md]: @matteogalliani are we open to evaluating other platforms?

File: personal-growth/learning-english.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+language. While they may seem intimidating at first, coaches are there to help you and are one of
+the best ways to expedite your learning.
+
+A lot of folks here at Nebulab have used and strongly recommend [italki](https://www.italki.com/),
+where you can choose your coach and interact with them remotely, but feel free to choose the online
+or offline platform that best suits you. [It's on us!](https://github.com/nebulab/playbook/blob/master/personal-growth/benefits.md)
+
+We recommend holding at least two 1-hour sessions per week with your coach, however this varies with
+your learning style and experience, so ask your mentor and they'll be able to provide tailored
+guidance.
```
Comment (kennyadsl) [inline personal-growth/learning-english.md]: I'd specify in this paragraph that it's ok to have lessons during working time, even if maybe we should also allow people to take lessons after work, like in the evening or on Saturday if it's more comfortable for them.

Comment (aldesantis) [reply personal-growth/learning-english.md]: @kennyadsl done!

**Pull request merged**